impala/tests/test_dbapi_connect.py

```diff
@@ -85,3 +85,22 @@ def test_impala_plain_connect(self):
     def test_hive_nosasl_connect(self):
         self.connection = connect(ENV.host, ENV.hive_port, timeout=5)
         self._execute_queries(self.connection)
+
+class ImpalaSocketTests(unittest.TestCase):
+    
+    def run_a_query(self):
+        with connect(ENV.host, ENV.port) as connection:
+            with connection.cursor() as cursor:
+                cursor.execute('select 1 as a_number')
+                return cursor.fetchall()
+    
```

Comment:
Can you remove the trailing whitespace here?

Reply:
Removed in 99f1d83!